aten/src/ATen/native/quantized/cpu/qnnpack_utils.cpp

```diff
@@ -0,0 +1,20 @@
+#ifdef USE_QNNPACK
+#include <ATen/native/quantized/cpu/qnnpack_utils.h>
```

Comment:
Shall we move the implementation to ATen/ParallelMobile.h (or some better name e.g. ParallelPThreadPool)? together with other ParallelOpenMP.h / ParallelNative.h / ParallelNativeTBB.h.

I think the reasons to not call it "qnnpack_xxx" are:
- NNPack on mobile should use this threadpool instead of 3rd-party/pthreadpool, right?
- We might want to provide parallel_for API (and other APIs defined in Parallel.h" for mobile. Probably should start with creating mobile friendly implementation there first, then hook up with the general parallel API.

Can we also follow the API convention there (use namespace instead of class + static methods)?

cc: @ilia-cher